src/main/java/com/iota/iri/storage/Persistable.java

```diff
@@ -2,10 +2,49 @@
 
 import java.io.Serializable;
 
+/**
+ * Any object that can be serialized and stored in the DB.
+ *
+ * @see Indexable
+ */
 public interface Persistable extends Serializable {
+    /**
+     * Serializes the core fields of the object to bytes that can be stored in a database. Core fields are all the
+     * fields that represent the object in other nodes in the network.
```

Comment:
"the object in other nodes" is quite vague, the other nodes likely also have metadata
```suggestion
     * fields that represent the core data required to construct this object.
```